src/Microsoft.TestPlatform.Extensions.BlameDataCollector/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -67,7 +67,7 @@ Microsoft.TestPlatform.Extensions.BlameDataCollector.ProcDumpArgsBuilder.BuildHa
 Microsoft.TestPlatform.Extensions.BlameDataCollector.ProcDumpArgsBuilder.BuildTriggerBasedProcDumpArgs(int processId, string filename, System.Collections.Generic.IEnumerable<string> procDumpExceptionsList, bool isFullDump) -> string
 Microsoft.TestPlatform.Extensions.BlameDataCollector.ProcDumpArgsBuilder.ProcDumpArgsBuilder() -> void
 Microsoft.TestPlatform.Extensions.BlameDataCollector.ProcDumpDumper
-Microsoft.TestPlatform.Extensions.BlameDataCollector.ProcDumpDumper.AttachToTargetProcess(int processId, string outputDirectory, Microsoft.TestPlatform.Extensions.BlameDataCollector.DumpTypeOption dumpType, bool collectAlways) -> void
+Microsoft.TestPlatform.Extensions.BlameDataCollector.ProcDumpDumper.AttachToTargetProcess(int processId, string outputDirectory, Microsoft.TestPlatform.Extensions.BlameDataCollector.DumpTypeOption dumpType, bool collectAlways, System.Action<string> logWarning) -> void
```

Comment:
Safe to do, I did it few times in the past as well.. This should not even be public api.